RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptionsViewModel.swift

```diff
@@ -0,0 +1,185 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  ManageSubscriptionsViewModel.swift
+//
+//
+//  Created by Cesar de la Vega on 27/5/24.
+//
+
+import Foundation
+import RevenueCat
+
+#if !os(macOS) && !os(tvOS) && !os(watchOS) && !os(visionOS)
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+@available(macOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+@MainActor
+class ManageSubscriptionsViewModel: ObservableObject {
+
+    @Published
+    var subscriptionInformation: SubscriptionInformation?
+    @Published
+    var refundRequestStatus: String?
```

Comment:
Nitpicking but should we rename to `refundRequestStatusMessage` or something like that since it has the actual message? Otherwise, we should change to some sort of enum I think